packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -11,15 +11,15 @@ import type {
   InterfaceContext,
 } from '@metamask/snaps-sdk';
 import type { JSXElement } from '@metamask/snaps-sdk/jsx';
-import { getJsonSizeUnsafe, validateJsxLinks } from '@metamask/snaps-utils';
+import {
+  getJsonSizeUnsafe,
+  getJsxInterface,
+  validateJsxLinks,
+} from '@metamask/snaps-utils';
 import { assert } from '@metamask/utils';
 import { nanoid } from 'nanoid';
 
-import {
-  constructState,
-  getJsxInterface,
-  validateInterfaceContext,
-} from './utils';
+import { constructState, validateInterfaceContext } from './utils';
```

Comment:
?

Reply:
Sorry I moved this function to use it in the simulator but ended up not using it. reverted